index.js

```diff
@@ -91,12 +92,13 @@ client.on("message", async (message) => {
 	}
 
 	// stops banned players
-	if (userProfile.account.banned) {
-		return message.reply("You are banned from Mega-RPG. You can plead for an unban at our support servers");
+	if (userProfile.account.banTime > Date.now()) {
```

Comment:
Vi kan godt ha det slik nå, selv om jeg i og for seg ikke er helt fan av det ettersom det gjør botten tregere siden hver eneste kommando som blir utført så må man lage en nye Dato.

Hvis ikke så kan dette være en alternativ løsning:
user.account.banned kan være enten false (0 eller null) eller en Dato. Da kan man bare sjekke om denne er true eller false for så å gjøre kalkulasjonene for om brukeren bør bli unbannet eller ikke hvis de er det

Reply:
Det var en banned = true/false før som sjekket om brukeren var bannet, men det funket ikke helt som vi ønsket. Så nå setter vi bare en tid for hvor lenge spilleren er bannet som øker for hver gang han ikke klarer å skrive inn captcha.
Det tar forsåvidt 0.002 ms å utføre en `Date.now()` som ikke er dyrt for botten å gjennomføre